BLSQ/iaso
# Remove redux part 1

Removing app, sidebar, users reducers.
Leaving org units and instances reducers for now

Related JIRA tickets : IA-3132

## Self proofreading checklist


## Changes

- replace app reducer to handle localisation with a context
- replace sidebar reducer with a context
- use react-query cache to get current user
- remove reducers, actions and useless dispatch

## How to test

Open the dashboard:
- check that all current user infos are correct
- open and close the sidebar 
- switch language , also on home page
- switch language also on setup account page (you need a super user not linked to an account)
- verify that embedded calendar and pages are working
